contents/docs/billing/estimating-usage-costs.mdx

```diff
@@ -144,7 +144,7 @@ We don't have data on those yet, unfortunately. In that case, we'd recommend goi
 Each product can be tuned to use only the resources you need. 
 
 - For product analytics, read [how to capture fewer unwanted events](/tutorials/fewer-unwanted-events)
-- For session replay, read [how to only record the sessions you want](/tutorials/limit-session-recordings)
+- For session replay, read [how to control which sessions you record](https://posthog.com/docs/session-replay/how-to-control-which-sessions-you-record)
```

Comment:
```suggestion
- For session replay, read [how to control which sessions you record](/docs/session-replay/how-to-control-which-sessions-you-record)
```

Reply:
@ivanagas Should we archive the tutorial and point that here too?

Reply:
we still get support requests that the tutorial covers

so I like the split of having a page that just explains the controls and then another that dives into them...

maybe we should even split the tutorial up more instead?